### ISSUE TITLE
GCC 4.6 compatibility

### ISSUE THREAD

**Issue by mlange05** [opesci/opesci-fd#62]
Ubuntu LTS and travis use GCC 4.6 as the default compiler, so we should aim to support it. We currently don't due to the use of non-static data member initializers, which is a C++11 feature and easy to change.

**Comment by marcoscimatec** [comment]
Shall we change it after the cgen code is merged?

**Comment by mlange05** [comment]
Yes, lets land PR #60 first. It will make this one easier, I think.
